uxarray/core/dataset.py

```diff
@@ -347,7 +345,13 @@ def nearest_neighbor_remap(
         coord_type : str, default="spherical"
             Indicates whether to remap using on spherical or cartesian coordinates
         """
-        return _nearest_neighbor_uxds(self, destination_obj, remap_to, coord_type)
+        warn(
+            "This implementation of using remapping is being removed in the near future. It is advised to use "
+            "'uxds.remap.nearest_neighbor'",
```

Comment:
"This implementation of performing remapping will be deprecated in a future release. It is advised to use `uxds.remap.nearest_neighbor()` instead."

Small rephrasing.